docs/install.rst

```diff
@@ -0,0 +1,210 @@
+.. inclusion-marker-start-do-not-remove
+
+Installation Guide
+==================
+
+Frameworks
+----------
+
+Horovod can be built with plugins for TensorFlow, PyTorch, and MXNet. By default, Horovod will attempt to build
+support for all of them. At least one must be enabled for Horovod to install successfully.
+
+Extras can be used when installing to enforce that framework dependencies are properly installed before attempting
```

Comment:
To ensure that framework dependencies are properly installed before attempting to install Horovod, append extra arguments that identify the required frameworks: